app/Hutch.Relay/Hutch.Relay.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
For a new project, let's use this opportunity for
`TreatWarningsAsErrors` to be `true` as well.